dataland-frontend/src/components/resources/dataTable/MultiLayerDataTable.vue

```diff
@@ -44,5 +50,6 @@ defineProps<{
   config: MLDTConfig<T>;
   mldtDatasets: Array<MLDTDataset<T>>;
   ariaLabel?: string;
+  hideEmptyFields: boolean;
```

Comment:
I assume this is no longer needed here? As well as the other changes in this file?